cobra-prompt.go

```diff
@@ -52,6 +52,9 @@ type CobraPrompt struct {
 
 	// InArgsParser adds a custom parser for the command line arguments (default: strings.Fields)
 	InArgsParser func(args string) []string
+
+	// SuggestionFilter will be uses when filtering suggestions as typing
+	SuggestionFilter func(completions []prompt.Suggest, sub string, ignoreCase bool) []prompt.Suggest
```

Comment:
By changing the String parameter to prompt.Document it gives the user of the library more flexibility to react on the user input. And since the ignoreCase is always true (ref line 169) I don't see the usecase for that parameter. So i suggest the following changes.
```suggestion
	SuggestionFilter func(suggestions []prompt.Suggest,  document *prompt.Document) []prompt.Suggest
```

Reply:
Yeah sounds good! i'll fix!